tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -502,6 +503,52 @@ def test_transformations(self):
             assert obj['payload']['content'] in self.FIRST_INPUT_LIST
             assert obj['payload'][ts_fieldname] == ts
 
+    @cluster(num_nodes=5)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_2), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_1), auto_create_topics=False, security_protocol=SecurityConfig.PLAINTEXT)
+    @cluster(num_nodes=5)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_10_0), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @parametrize(broker_version=str(LATEST_0_9), auto_create_topics=True, security_protocol=SecurityConfig.PLAINTEXT)
+    @cluster(num_nodes=6)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=False, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(DEV_BRANCH), auto_create_topics=True, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=False, security_protocol=SecurityConfig.SASL_SSL)
+    @parametrize(broker_version=str(LATEST_0_11_0), auto_create_topics=True, security_protocol=SecurityConfig.SASL_SSL)
```

Comment:
Hmm, are security configs really relevant here if we don't do anything with ACLs? Seems like a lot of parameterizations here and we already get coverage from the `test_file_source_and_sink`?

In general we've started to be more careful about a ton of parameterizations and covering more within each test since the setup/teardown costs can be quite substantial.